README.md

```diff
@@ -54,9 +54,27 @@ Total: 10, Dialing: 1, Established: 9, Closed: 0, Error: 0, NotInitiated: 0
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
 --- myhttpsamplehost.com:80 tcp test statistics ---
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
+Time spent in 10 established connections min/avg/max/stdDev = 17.929ms/19.814ms/29.811ms/3.353ms
```

Comment:
What about "Per-connection timings to successfully dial X connections". "Spent" has a negative meaning. And, as you are saying "in X established connections", it may sound as the total time, not individualised.